packages/api/lib/response.js

```diff
@@ -281,9 +281,7 @@ function handle(event, context, authCheck, func) {
       if (!obj.expires) return cb('Invalid Authorization token');
       if (obj.expires < Date.now()) return cb('Session expired');
       return func(cb);
-    }).catch((e) => {
-      cb('Invalid Authorization token', e);
-    });
+    }).catch(cb);
```

Comment:
Would this kind of change need to be documented in the CHANGELOG? It would potentially have an impact on users. For authorization errors, the lambda response would still return the same errors as before (as the tests show). However, this change means that other errors, like a bad input payload, will now be properly included in the lambda response.

Reply:
To clarify my "reasonable response" comment:

The acceptance criteria of CUMULUS-917 says "When posting to the API endpoint for creating a new provider, it returns a 401/403 error" and "When posting to the API endpoint for creating a new collection, it returns a 401/403 error".  

1. Write a test that tries to create a new provider without a valid auth token and make sure it returns a 401.
2. Write a test that tries to create a new collection without a valid auth token and make sure it returns a 401.
3. Make sure that all of the other endpoint tests work.

Reply:
I think you already wrote tests that cover the 401/403 cases:
https://github.com/nasa/cumulus/blob/CUMULUS-916-917/packages/api/tests/endpoints/providers.js#L149

But given that 917 says `When posting to the API endpoint for creating a new provider, it does not create a provider`, I can add assertions that no provider/collection is created by checking Dynamo, as you suggested below.